keras_cv/layers/preprocessing/random_shear.py

```diff
@@ -151,3 +156,167 @@ def get_config(self):
             }
         )
         return config
+
+    def augment_bounding_boxes(self, image, bounding_boxes, transformation=None):
```

Comment:
Remove image argument, keep bounding_boxes argument.  Add **kwargs

PR https://github.com/keras-team/keras-cv/pull/493 updated this.